OpenAnatomyExport/OpenAnatomyExport.py

```diff
@@ -420,7 +420,9 @@ def addModelToRenderer(self, inputModelNode, outputModelNode, boostGouraudColor=
       self._temporaryExportNodes.append(self._decimationParameterNode)
 
     # Quadric decimation
-    if (self.reductionFactor == 0.0) or (inputModelNode.GetPolyData().GetNumberOfPoints() < 50):
+    if ((self.reductionFactor == 0.0) or (inputModelNode.GetPolyData().GetNumberOfPoints() < 50) 
```

Comment:
Add to the comment why we skip decimation if there are lines or vertices in the polydata.